script/countries.py

```diff
@@ -0,0 +1,27 @@
+"""Helper script to update country list.
+
+ISO does not publish a machine readable list free of charge, so the list is generated
+with help of the pycountry package.
+"""
+from pathlib import Path
+
+import pycountry
+
+from .hassfest.serializer import format_python_namespace
+
+countries = {x.alpha_2 for x in pycountry.countries}
```

Comment:
This is a set. How do you guarantee it's sorted?

Reply:
`format_python_namespace` automagically sorts sets.

https://github.com/home-assistant/core/blob/7df711f1f3208476ee604125e475fba7f95dd653/script/hassfest/serializer.py#L50-L51